### PR TITLE
[update] windows revere script

### PR DESCRIPTION
**Pull request by NatLee**
更新Windows上reverse的script

主要是把SSH相關設定拉到腳本前面 :D

### CODE REVIEW

File: src/tunnels/views.py [hunk 1]
```diff
@@ -250,12 +257,14 @@ def get_script(
     public const uint ES_DISPLAY_REQUIRED = 0x00000002;
 }}
 "@
+
```
Comment (h-alice) [inline src/tunnels/views.py:260]: Blank Line : D

File: src/tunnels/views.py [hunk 2]
```diff
@@ -236,8 +236,15 @@ def get_script(
     ):
         reverse_port = server_auth_key.reverse_port
 
-        config_string = f"""$continue = $true
-echo "[+] Script started"
+        config_string = f"""echo "[+] Script start"
```
Comment (h-alice) [inline src/tunnels/views.py:239]: Embed script (powershell) in a completely different language (Python) can cause chaos and hard to maintain.
I'll make a PR to implement template system for this issue, so it's OK for now.

**Review by h-alice — APPROVED** [review]
I have made a review about embedding different language in Python. https://github.com/NatLee/telepy/blob/c0b600db8916e9c8662fb799381c96bcc3a5d053/src/tunnels/views.py#L225
It's obviously not a good practice, and I'm doing research about template system for this issue, so it's OK for now.

Everything is OK and ready to be merged. 🚀

**Comment by NatLee** [comment]
Seems Okay now, go!

**Pull request merged**